src/components/ecology.jsx

```diff
@@ -3,42 +3,44 @@ import API from "./api";
 import Overview from "./overview";
 
 export default class Ecology extends React.Component {
-  renderAPI(source) {
```

Comment:
One benefit of passing `this.props.source` as an argument to `renderAPI` is ease of testing. You can just call `FooClass.prototype.methodName` directly in your test (I think that still works with ES6 `class` sugar).